monitoring/monitoring.go

```diff
@@ -362,3 +362,27 @@ func newCadvisor(env e2e.Environment, name string, cgroupPrefixes ...string) *In
 		Privileged: true,
 	}), "http")
 }
+
+const nginxImage = "docker.io/nginx:1.21.1-alpine"
+
+// NewStaticMetricsServer creates a new nginx server that serves the content of metrics as /metrics endpoint.
+// This is useful for testing different metrics scrapers.
+func NewStaticMetricsServer(e e2e.Environment, name string, metrics []byte) *InstrumentedRunnable {
```

Comment:
This is great! I feel like the functionality this provides, goes beyond just metrics scrape. 
This can be used for quite a few things apart from Prometheus scraping, like for example serving webpage to web scraper tool or something that polls remote config.

Maybe we can make it a bit generic, and put this in its own package? We can document the metrics scraping as one possible use case then. 🙂 
```suggestion
func NewStaticServer(e e2e.Environment, name, filename, endpoint string, content []byte) *InstrumentedRunnable {
```

Reply:
I think it's fine to focus on one small specific aspect for now. We can review later before 1.0 and see if other use cases pop up (: